src/qfieldcloud_sdk/sdk.py

```diff
@@ -117,21 +118,35 @@ def logout(self) -> None:
         return resp.json()
 
     def list_projects(
-        self, username: Optional[str] = None, include_public: Optional[bool] = False
-    ) -> Dict:
-        """Lists the project of a given user. If the user is omitted, it fallbacks to the currently logged in user"""
-        resp = self._request(
-            "GET",
-            "projects",
-            params={
-                "include-public": "1" if include_public else "0",
-            },
-        )
+        self,
+        username: Optional[str] = None,
+        include_public: Optional[bool] = False,
+        limit: Optional[int] = None,
+        offset: Optional[int] = None,
+    ) -> List[Dict[str, Any]]:
+        """Returns a paginated lists the project of a given user. If the user is omitted, it fallbacks to the currently logged in user"""
+        params = {"include-public": "1" if include_public else "0"}
```

Comment:
```suggestion
        params = {
            "include-public": int(include_public),
        }
```

I think it is safe to convert the `Optional[bool]` to `int` like this. No idea why initially didn't write it this way.

Reply:
```python
int(False) # ok
int(True) # ok
int(None) # TypeError
```